azure-pipelines.vrt-baseline.yml

```diff
@@ -15,50 +15,54 @@ variables:
 pool: '1ES-Host-Ubuntu'
 
 jobs:
-  - job: VRToolUpdateBaseline
+  - job: VRToolUpdateBaseline_V9
     workspace:
       clean: all
     steps:
       - template: .devops/templates/tools.yml
 
-      - task: Bash@3
-        inputs:
-          filePath: yarn-ci.sh
-        displayName: yarn (install packages)
+      - template: .devops/templates/runpublishvrscreenshot.yml
+        parameters:
+          fluentVersion: v9
+          storyPackageName: '@fluentui/vr-tests-react-components'
+          storyPackagePath: 'apps/vr-tests-react-components'
+          shouldbuildstorybookaddon: true
 
-      - script: |
-          yarn workspace @fluentui/vr-tests-react-components convert
-        displayName: Convert screener component to storywright
-
-      - script: |
-          yarn workspace @fluentui/vr-tests-react-components screener:build
-        displayName: Build VR tests components package
-
-      - script: |
-          yarn lage test:component --verbose
-        displayName: 'Run VR tests'
-
-      - script: |
-          mkdir -p screenshots
-          cp -rf apps/vr-tests-react-components/dist/screenshots/*.png screenshots/
-        displayName: Collate Artifacts
+      - bash: node node_modules/vrscreenshotdiff/lib/index.js release --clientType "fluentuiv9" --buildId $(Build.BuildId)
```

Comment:
Just for context, can you explain what this step is doing? And are all the secrets being referenced from the `fluent-vr-tool` Azure resource group?

Reply:
This step is responsible to 
For baseline pipeline
1. Upload the screenshots to azure blob storage
2. Add screenshot metadata in cosmos database like buildid, pipelineid, screenshotpath in blob storage.
For pr pipeline 
1. Download baseline screenshots
2. Diff baseline screenshot with pr screenshots
3. upload diffed screenshots to blob storage
4. Add details about diffing result in cosmos database
5. Post comment on PR

Currently the secret are stored as secret variables in pipeline. The resource at the moment are pointing to our tests resurce group. As soon as piyush is back from sick leave we will point to resources in fluent-vr-tool resource group